### ISSUE TITLE
[TOPIC] Building a GitHub activity dashboard on your project with Airbyte and Metabase

### ISSUE THREAD

**Issue by johnlafleur** [airbytehq/write-for-the-community#29]
Inspiration: https://preset.io/blog/2020-04-15-github-community-dashboard/

Metrics we'd like to see:
- evolution of the # stargazers
- evolution of the # watchers
- evolution of the # forks
- evolution of the # issue contributors
- evolution of the # PR contributors
- evolution of the # new comments from non-team members
- evolution of the # new issues from non-team members
- evolution of the # new PRs from non-team members
- time to 1st comment for non-team members interactions (50th, 90th percentile)
- time until 1st review for a PR (50th, 90th percentile)
- time until merge for a PT (50th, 90th percentile)

**TO PROMOTE:**
- [ ] Opensource.com - by @johnlafleur 
- [ ] Towards Data Science - by @johnlafleur 
- [ ] dev.to - by @johnlafleur 
- [ ] Hackernoon - by @johnlafleur 
- [ ] Dzone - by @johnlafleur

**Comment by ChristopheDuong** [comment]
THese two issues are a bit early to make an article maybe?

You have airbyte at the start of the data pipeline and either preset or metabase at the other end.
But you might need a tiny bit of transformation in between the two.... If we could work out more our transformation story with migrations, customizable transformations etc, it would be easier for these stories

**Comment by johnlafleur** [comment]
After some discussion with Michel, we decided to move forward with this tutorial. We will revamp along the way with our progress on transformations. We need to understand that each new release will be a trigger to check if we need to revamp some tutorials. This will be the case for this one with transformation. 

Here is a Google Slide to illustrate how the dashboard could look like: 
https://docs.google.com/presentation/d/1SkY4ID5akCw1qCK1ioKsWdweKO_RcaGFfS5w8TpP0o0/edit?usp=sharing

**Comment by DominusKelvin** [comment]
Got it @johnlafleur

**Comment by DominusKelvin** [comment]
@johnlafleur in the last metric, you mean PR right?

**Comment by johnlafleur** [comment]
Yes indeed. But please use the Google Slides (https://docs.google.com/presentation/d/1SkY4ID5akCw1qCK1ioKsWdweKO_RcaGFfS5w8TpP0o0/edit?usp=sharing) as the source of truth :)

**Comment by DominusKelvin** [comment]
Got it @johnlafleur

**Comment by avaidyanatha** [comment]
@DominusKelvin What's the status on this one?

**Comment by avaidyanatha** [comment]
Awesome, thanks!

**Comment by DominusKelvin** [comment]
> @DominusKelvin What's the status on this one?

Hey I believe this one is done now.
